chainer/functions/vae.py

```diff
@@ -0,0 +1,106 @@
+import math
+
+import chainer.functions as F
+from chainer import variable
+
+
+def gaussian_kl_divergence(z_mean, z_ln_var):
+    """Calculate KL-divergence between a given gaussian and the normal one.
```

Comment:
Normal distribution is an alias of Gaussian distribution. So we should call `N(0, 1)` standard normal distribution